javascript/promisify-a-callback.mo.md

```diff
@@ -0,0 +1,64 @@
+# [MO] Promisify a callback _(~5 min)_
+
+## Owner: Arthur Levoyer
+
+## Why
+
+When asynchronous action is performed, if you want to wait for its success or failure and in order to avoid several chains of callbacks. See [callback hell](http://callbackhell.com/).
```

Comment:
When **an** asynchronous... ...if you want to wait for its success or failure and avoid several chains of callbacks